aws/src/test/java/org/apache/iceberg/aws/glue/TestGlueCatalog.java

```diff
@@ -144,6 +144,32 @@ public void testDefaultWarehouseLocationDbUri() {
     Assert.assertEquals("s3://bucket2/db/table", location);
   }
 
+  @Test
+  public void testDefaultWarehouseLocationCustomCatalogId() {
```

Comment:
The integration tests in `TestGlueCatalogNamespace` also don't set the catalogId, do we need to set it in those files also?

Reply:
Are there multiple AWS accounts available for integration tests?  
I could not find that setup either in `AwsIntegTestUtil` or `TestAssumeRoleAwsClientFactory` (as and obvious potential cross account example).

Reply:
I presume it is a single account. 
The Integration testcase can be executed by exporting the mentioned environment variables (https://github.com/apache/iceberg/pull/4855)

Thanks for the clarification. may be no need of catalog id for these tests.